examples/zkapps/04-zkapp-browser-ui/ui/src/styles/globals.css

```diff
@@ -47,8 +47,3 @@ a {
   color: inherit;
   text-decoration: none;
 }
-
-a:hover {
```

Comment:
Nice catch, I think because I was using default global.css for UI: https://github.com/o1-labs/zkapp-cli/blob/main/src/lib/ui/next/styles/globals.css
I'll revert this. Thank you!